db/models/users.js

```diff
@@ -0,0 +1,13 @@
+const db = require("../connection");
+
+exports.fetchUsers = () => {
+
+	const queryStr = `SELECT * FROM users;`;
+
+	return db.query(queryStr).
+	then(results => {
+		if (results.rows.length === 0) {
+			return Promise.reject({ status: 404, msg: "no users found" });
```

Comment:
Just thinking to the front end, i'd suggest handling this with just an empty array. 
You could then handle this on the front to display this message if the array is empty.